modules/silicon_design/scripts/build/images/provision/environment.yml

```diff
@@ -34,10 +34,55 @@ dependencies:
   - pandas
   - pymeep=*=mpi_mpich_*
   - jupyterlab
-  - python
+  - litex-hub::gcc-riscv32-elf-newlib
```

Comment:
you shoudn't need to prefix them by `litex-hub` if it's the first one in the list of channels.

Reply:
if we want to prefix those, then we should do it consistently for all packages.

Reply:
I'll remove these prefixes